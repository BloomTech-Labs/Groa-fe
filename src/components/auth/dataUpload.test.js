import React from "react";
import { render } from "@testing-library/react";
import DataUpload from './dataUpload';
import { clikDataUpload } from "../../utils/test-utils.js";

describe('dataupload', () => {
    it('dataupload be defined', () => {
      expect(DataUpload).toBeDefined();
    });
  
    it('should render dataupload correctly', () => {
      const tree = (
        <DataUpload name='dataupload file test' />
      );
      expect(tree).toMatchSnapshot();
    });
   });
  
   test('drop dataupload file at upload spot', () => {
      expect(clikDataUpload.sum(0, 1)).toBe(1)
  })