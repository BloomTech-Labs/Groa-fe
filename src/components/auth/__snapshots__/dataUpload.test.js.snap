// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button postlogin should render postlogin correctly 1`] = `
<postLogin
  name="post login button test"
/>
`;

exports[`dataupload should render dataupload correctly 1`] = `
<DataUpload
  name="dataupload file test"
/>
`;
